Apps/frontend/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN npm install
 COPY . ./
 
 # Expose vite port to host machine
-RUN npm run build -- --mode ${MODE}
+RUN npm run build --mode ${MODE}
```

Comment:
Could this be the reason why the build fails?

Reply:
No, this was the workaround Philipp suggested so that it does not fail anymore.